src/foam/nanos/crunch/ui/CapableMinMaxCapabilityWizardlet.js

```diff
@@ -104,34 +133,42 @@ foam.CLASS({
       code: function(oldChoices, newChoices) {
         var choiceWizardlets = this.choiceWizardlets;
 
-        var selectedOldChoices = oldChoices.filter(choice => typeof choice[2] === 'object' ? choice[2].get() : choice[2]);
-        var selectedNewChoices = newChoices.filter(choice => typeof choice[2] === 'object' ? choice[2].get() : choice[2]);
+        var selectedOldChoicesNames = oldChoices.map(choice => choice[0]);
+        var selectedNewChoicesNames = newChoices.map(choice => choice[0]);
```

Comment:
non-blocking review comment; soon we should re-evaluate using the names this way. Probably the choices should have an ID which is obtained from the Capability's ID